### PR TITLE
Remove atomic usage for RG32Uint.

### PR DESCRIPTION
**Pull request by squidbus**
Looking into https://github.com/KhronosGroup/MoltenVK/issues/2414, it appears that 64-bit atomics for RG32 texture is not supported by Vulkan at this time, and code for it was only added to prepare for if it is in the future.

However, marking every texture with this format as atomic not only could have performance implications, it is also causing issues on devices like M1 family GPUs that support native texture atomics but not 64-bit atomics. Thus it is better to remove it and revisit if Vulkan ever does decide to support this.

I left the part enabling MTLPixFmtCaps as it already has a flag to force-disable it since it isn't supported by Vulkan along with a comment explaining, and thus it seems pretty harmless. Although I removed enabling it on `RG32Sint` as according to the spec Metal only supports it for `RG32Uint`.

### CODE REVIEW

File: MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm [hunk 1]
```diff
@@ -1542,7 +1542,6 @@
 	// Including this here so we remember to update this if support is added to Vulkan in the future.
 	bool atomic64 = noVulkanSupport && (gpuCaps.supportsApple9 || (gpuCaps.supportsApple8 && gpuCaps.supportsMac2));
 	enableMTLPixFmtCapsIf( atomic64, RG32Uint, Atomic );
-	enableMTLPixFmtCapsIf( atomic64, RG32Sint, Atomic );
```
Comment (cdavis5e) [inline MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm:1545]: Why did you delete one line, but not the other?

**Review by cdavis5e — CHANGES_REQUESTED** [review]

File: MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm (hunk 1 above)
Comment (squidbus) [reply MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm:1545]: Mentioned this in the description, this part being left here seems benign since it's disabled by a bool and has a comment that its left as a reminder, but according to the Metal spec 64-bit texture atomics are only supported for RG32Uint, not RG32Sint, so that had to be removed regardless.

Comment (squidbus) [reply MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm:1545]: I can remove the whole section if you want though, since it's not being used.

Comment (cdavis5e) [reply MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm:1545]: Ah, OK. Silly me. You can leave it in.

**Review by cdavis5e — APPROVED** [review]
